src/main/resources/embedding-providers-config.yaml

```diff
@@ -104,6 +104,47 @@ stargate:
                   help: "Vector dimension to use in the database and when calling Azure OpenAI."
             - name: text-embedding-ada-002
               vector-dimension: 1536
+        bedrock:
+          display-name: Amazon Bedrock
+          enabled: true
+          url: " "
```

Comment:
This needs to be handled separately as URL field is mandatory in data-api and there is no endpoint url available for bedrock since we use the aws sdk.